module/move/willbe/src/cargo.rs

```diff
@@ -43,11 +43,13 @@ mod private
   }
   
  /// Upload a package to the registry
-  pub fn publish< P >( path : P, dry : bool ) -> Result< CmdReport >
+  pub fn publish< P, Pb >( path : P, dry : bool, temp_dir : Option< Pb > ) -> Result< CmdReport >

```

Comment:
Can we modify the function so that it accepts an object similar to `TestArgs`, how the cargo::test does? Or what do you think about it?